packages/profiling-node/src/cpu_profiler.ts

```diff
@@ -172,6 +172,12 @@ class Bindings implements V8CpuProfilerBindings {
       return;
     }
 
+    if (typeof PrivateCpuProfilerBindings.startProfiling !== 'function') {
```

Comment:
I do not know for sure, but PrivateCpuProfilerBindings could also be falsy, so this would throw before getting to the typeof check. Would be safer if we added nullish coalescing here
```suggestion
    if (typeof PrivateCpuProfilerBindings?.startProfiling !== 'function') {
```

Reply:
We check against this right above though. I was thinking about combining the checks but I think it makes sense to have different error messages and I think bundle size isn't the biggest concern here.

Reply:
My bad, Github preview did not show that line and I did not double check. Thanks for making these changes!